projects/gson/build.sh

```diff
@@ -15,8 +15,11 @@
 #
 ################################################################################
 
-MAVEN_ARGS="-Dmaven.test.skip=true -Djavac.src.version=11 -Djavac.target.version=11 -X"
-$MVN --batch-mode --update-snapshots verify ${MAVEN_ARGS}
+MAVEN_ARGS="-DskipTests -Dmaven.compiler.release=8 -X"
+# Only build 'gson' Maven module
+cd gson
```

Comment:
Could also alternatively run Maven for the parent project and only build the `gson` module as described in https://stackoverflow.com/a/3899772, though this approach works as well.
Do you prefer one solution over the other?